src/components/dropdowns/dropdowns.js

```diff
@@ -40,7 +40,7 @@ var dh2oDropdown = function (dropdown, autoShow=true, callback=null) {
   var content = dropdown.getElementsByClassName(contentStr)[0]
   dropdown.show = function () {
     if (content) {
-      if (callback) { callback(true) }
+      if (callbackFunc) { callbackFunc(true) }
```

Comment:
Why rename this? Isn't it being a function sort of implied by being a callback?

Reply:
eslint rule: no-callback-literal